src/components/common/SearchPannel/SearchPannel.jsx

```diff
@@ -0,0 +1,19 @@
+import './searchPannel.scss';
+
+const SearchPannel = () => {
+  return (
+    <form className="search-container" action="/search" method="GET">
+      <button type="submit" className="search-button">
+        <img src="../../../src/assets/img/search-icon.png" alt="Search icon" />
+      </button>
+      <input
+        type="text"
```

Comment:
It's better to use existing type `search`

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/search